libraries/standard/mqtt/src/iot_mqtt_helper.c

```diff
@@ -25,6 +25,9 @@
  * @brief Implements helper functions for the MQTT library.
  */
 
+/* The config header is always included first. */
+#include "iot_config.h"
```

Comment:
Why do we need iot_config.h in this file ( since you have added AWS_IOT_MQTT_ENABLE_METRICS below )?

Reply:
I included it here since logging and assert macros were used in this file, which are configured in iot_config.h. Including it has no impact on MISRA though, so I will remove it